pkgs/applications/video/dmlive/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, pkg-config
+, makeWrapper
+, openssl
+, mpv
+, ffmpeg
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dmlive";
+  version = "unstable-2022-06-05";
+
+  src = fetchFromGitHub {
+    owner = "THMonster";
+    repo = pname;
+    rev = "f57bfaefa91e069cc154e66bdb3993291711bd51";
+    hash = "sha256-nr5+UIj6/3BkZRPz8fOQBws+595LDSZVJBvc3K4PyXw=";
+  };
+
+  cargoHash = "sha256-Mkg/Ifr2mJRs97b+Dw6I1RuqXERpSigDEzGkr+wWN5w=";
+
+  OPENSSL_NO_VENDOR = true;
+
+  nativeBuildInputs = [ pkg-config makeWrapper ];
+  buildInputs = [ openssl ];
+
+  postInstall = ''
+    wrapProgram "$out/bin/dmlive" --prefix PATH : "${lib.makeBinPath [ mpv ffmpeg ]}"
```

Comment:
`nodejs` is also needed, see https://github.com/THMonster/dmlive/blob/fd4fa1859f05350658db598a50d29f59d22b55a1/src/utils/mod.rs#L32